docs/operators/csvParserOperator.md

```diff
@@ -0,0 +1,107 @@
+Csv Parser Operator
+=============
+
+## Operator Objective
+This operator is designed to parse delimited records and construct a map or concrete java class also known as ["POJO"](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) out of it. User need to provide the schema to describe the delimited data. Based on schema definition the operator will parse the incoming record to object map and POJO.  User can also provide constraints if any, in the schema. The supported constraints are listed in [constraints table](#constraints). The incoming record will be validated against those constraints. Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message.
+
+**Note**: field names of POJO must match field names in schema and in the same order as it appears in the incoming data.
+
+## Overview
+The operator is **stateless**, **idempotent**, **fault-tolerant** and **partitionable**.
+
+
```

Comment:
Could you add operator information section? That way we have concistency across all operator documentation. 
You may refer to 
https://github.com/shubham-pathak22/incubator-apex-malhar/blob/91f0ef5c57844e77ba1e7992e77d5da87415691a/docs/operators/jsonParser.md